ppl/src/test/java/org/opensearch/sql/ppl/antlr/PPLSyntaxParserTest.java

```diff
@@ -268,5 +272,56 @@ public void testDescribeCommandWithSourceShouldFail() {
 
     new PPLSyntaxParser().parse("describe source=t");
   }
+
+  @Test
+  public void testCan_parse_extract_function() {
```

Comment:
Unfortunately there's a mix of camel case and snake case between different files. However, we should try to make it consistent within a file. I think the names should be camel case based on already existing test names.